commands/command_parser.py

```diff
@@ -20,9 +20,13 @@ class CommandParser:
     
     @classmethod
     def parse_command(cls, bot_query: BotQuery) -> BotCommand:
+        if bot_query.has_callback_query:
+            cmd_name = bot_query.callback_query.callback_data.rsplit('/')[-1].lower()
```

Comment:
Can you either make `.rsplit('/')[-1].lower()` into a separate function or comment what this is doing?

Something like:
```
# Returns the case insensitive command name from the query callback data. Eg. `/command` -> `command`
```